force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Apex extra: Breach of naming conventions
#### Severity
Warning

#### Finding
Method `getSObjectAPINameBasedOnId` does not match naming conventions. Consider renaming (hint: `myMethod`).

#### Why is this a problem?
Naming convention help drive consistency across the codebase, which makes it easier to read and understand source code.

#### Help and documentation

* [Improper Adherence to Coding Standards (CWE)](https://cwe.mitre.org/data/definitions/710.html)